### DISCUSSION TITLE
Read Duco Box info -  Energy and filter time

### DISCUSSION THREAD

**Discussion by QMartens** [TheWalkingDadNL/ducoboxHTTP#3] [Q&A] [answered]
Hi all,

I've already initiated a flow to read out all of the nodes in my Duco system, thanks to this node-red contribution
However I'm not sure how to read out the Duco Box information that is not related to a specific node (device ID), and is not in the json of the box node itself. 
For example energy related and fan related values, or even more specific the parameter FilterRemainingTime.
Any help would be very appreciated!

**Comment by TheWalkingDadNL** [comment]
Does the example flow help? If you set the ducoboxHTTP node on the right deviceID you should recevice all values from that node.

**Reply by QMartens** [reply]
Hi @TheWalkingDadNL,

Thanks for the prompt reply!
I do read out all of the nodes, including the Duco Box itself on Node ID 1 (it is a DucoBox Energy), according on your example which was very usefull. All of the below listed values (from http://duco001.local/boxinfo.html) are indeed in the respective resulting jsons:
![image](https://github.com/TheWalkingDadNL/ducoboxHTTP/assets/61286657/38e89512-4324-4f45-b10c-66a3d6d41221)
However, some values from the Box info tab I cannot read from these nodes apperently:
![image](https://github.com/TheWalkingDadNL/ducoboxHTTP/assets/61286657/cd054486-6615-4595-887d-e7d04670a193)

And these parameters seem also very usefull to gather. For example to get an alert when the filters almost need to be replaced, to monitor the energy consumption, effectiveness of the heat pump (air temperatures in and out), etc.
Is it possible to read these with the current implementation?

**Reply by TheWalkingDadNL** [reply]
You have much more info in the "box info" tab then me. What ducobox do you have? Anyway, can you conform for me that all info you need are shown in this URL: http://duco001.local/boxinfoget

**Reply by QMartens** [reply]
Yes, all the info seems to be in the url:
![image](https://github.com/TheWalkingDadNL/ducoboxHTTP/assets/61286657/18db6f86-718b-4b54-a550-5f4de1f9ff44)
I have a DucoBox Energy 400 with 2 zones.

**Comment by TheWalkingDadNL** [comment, marked as answer]
I have published a new version (0.2.0). Can you update your node and check if you can fetch all the info you need? Please, because i dont use the boxinfo settings, give me feedback.

**Reply by QMartens** [reply]
Hi @TheWalkingDadNL,

Your solution works nicely alle the info is there, the checkbox is a great addition!
Many thansk for this update, and have a great new year ;-)
